src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx

```diff
@@ -35,7 +37,7 @@ export const SubmitArtworkAddDetails = () => {
               onChangeText={(e) => setFieldValue("year", e)}
               accessibilityLabel="Year"
               style={{ width: "50%" }}
-              autoFocus
+              autoFocus={currentStep === "AddTitle"}
```

Comment:
Here we need to check if the current step is the "AddDetails" step, right?
```suggestion
              autoFocus={currentStep === "AddDetails"}
```

Reply:
A comment would be great as well. Maybe something like:

```suggestion
              // Only focus on the input and toggle the keyboard if this step is visible to the user.
              autoFocus={currentStep === "AddTitle"}
```